Source/Core/Core/ConfigManager.h

```diff
@@ -20,7 +20,7 @@ enum class Region;
 }
 
 // DSP Backend Types
-#define BACKEND_NULLSOUND _trans("No audio output")
+#define BACKEND_NULLSOUND _trans("No Audio Output")
```

Comment:
@degasus, is this going to cause any problems? Breaking INI compatibility? (Or do we not care about INI compatibility for this choice?)